cmd/syncthing/main.go

```diff
@@ -376,13 +372,12 @@ func syncthingMain() {
 		runtime.GOMAXPROCS(runtime.NumCPU())
 	}
 
-	events.Default.Log(events.Starting, map[string]string{"home": confDir})
+	events.Default.Log(events.Starting, locations)
```

Comment:
Yep, true. I'll add it to the REST docs. I think this is safe enough as we're anyway in breaking-the-API mode at the moment.